src/components/Button.tsx

```diff
@@ -63,7 +63,7 @@ interface IconButtonProps {
 export const IconButton = forwardRef<HTMLButtonElement, IconButtonProps & ComponentProps<typeof BaseButton>>(
   function IconButton({ icon: Icon, iconProps, ...props }: IconButtonProps & ComponentProps<typeof BaseButton>, ref) {
     return (
-      <SecondaryButton {...props} ref={ref}>
+      <SecondaryButton {...props} ref={ref} style={{ transform: 'translateY(1.75px)' }}>
```

Comment:
put this in a `styled` wrapper instead of doing the inline style

nit - is it possible to accomplish this with some other tool? translate can be a little unexpected. It's better to use margin/padding/positioning in most cases. Not important to figure this out if you're busy.